webclient/pas/filters.py

```diff
@@ -404,7 +391,7 @@ class SamostatnyNalezFilterFormHelper(crispy_forms.helper.FormHelper):
                     "historie_datum_zmeny_od", css_class="col-sm-4 app-daterangepicker"
                 ),
                 Div("historie_uzivatel", css_class="col-sm-3"),
-                Div("historie_zapsal_uzivatel_organizace", css_class="col-sm-3"),
```

Comment:
@pesikj Zde jen upozorňuji na to, že funkce ve filtračním formuláři má být stejná jako jinde (podle `historie_uzivatel_organizace`). Pouze volba "Naše záznamy" se bude chovat jinak, tj. podle `projekt_organizace`.